src/main/java/seedu/address/logic/parser/ArgumentTokenizer.java

```diff
@@ -32,15 +36,33 @@ public ArgumentTokenizer(Prefix... prefixes) {
     }
 
     /**
-     * @param argsString arguments string of the form: preamble <prefix>value <prefix>value ...
+     * @param argsString arguments string of the form: preamble <prefix>value <prefix>value ... for add command
      */
     public void tokenize(String argsString) {
         resetTokenizerState();
-        List<PrefixPosition> positions = findAllPrefixPositions(argsString);
+        positions = findAllPrefixPositions(argsString);
         extractArguments(argsString, positions);
     }
 
     /**
+     * Create a floating task according to parameters entered
+     */
+    public void createFloatingTask() {
```

Comment:
This method reduces coherence in this class since `ArgumentTokenizer` is only meant to split a command into its respective `tokens`.